docs/how-to/pin-files.md

```diff
@@ -12,13 +11,46 @@ While this is nice, sometimes you want to be able to control what you keep aroun
 
 Let's look at this example to explore pinning to your local IPFS node in a bit more depth:
 
-```
-echo "ipfs rocks" > foo
-ipfs add foo
-ipfs pin ls --type=all
-ipfs pin rm <foo hash>
-ipfs pin rm <foo hash>
-ipfs pin ls --type=all
+1. First, create a file called `foo` with the contents `ipfs rocks`:
```

Comment:
Each tasks comes in a numbered item in a list. Each item starts with `1.`, even if it isn't the first item. Markdown automagically figures out how many items are in the list when it gets converted into HTML, so we don't have to bother writing in the numbers manually.

If a numbered line is followed by a code block (\`\`\`) then this line should end in a colon `:`.

File name and variables, like `foo` and `ipfs rocks`, are surrounded by back-ticks (\`).